spring-boot-modules/spring-boot-basic-customization-2/src/main/java/com/baeldung/enummapping/enums/Level.java

```diff
@@ -0,0 +1,7 @@
+package com.baeldung.enummapping.enums;
+
+public enum Level {
+
+    LOW, MEDIUM, HIGH;
```

Comment:
We can remove the semicolon is not necessary.

Reply:
done